tests/worksheets/src/test/scala/tests/worksheets/WorksheetSuite.scala

```diff
@@ -372,6 +372,22 @@ class WorksheetSuite extends BaseSuite {
        |""".stripMargin
   )
 
+  checkDecorations(
+    "akka".tag(SkipScala3).tag(SkipScala211),
+    """|import $dep.`com.typesafe.akka::akka-actor:2.6.13`
+       |import akka.actor.ActorSystem
+       |
+       |implicit val system = ActorSystem("worksheet")
```

Comment:
I minimized the test case. Creating an actor system should pick up the default `reference.conf` from the classpath, which doesn't work without the changes.